xbmc/Application.cpp

```diff
@@ -3210,7 +3210,8 @@ PlayBackRet CApplication::PlayStack(const CFileItem& item, bool bRestart)
       }
     }
 
-    *m_itemCurrentFile = item;
+    CFileItemPtr tmp(new CFileItem(item));
```

Comment:
IIRC there was a reason for deep copy into a new CFileItem object because the properties get altered in some cases